v0/destinations/customerio/transform.js

```diff
@@ -100,7 +101,16 @@ function responseBuilder(message, evType, evName, destination) {
         Math.floor(new Date(message.originalTimestamp).getTime() / 1000)
       );
     }
-
+    if (getFieldValueFromMessage(message, "timestamp")) {
```

Comment:
We're running the same loop again. Should we reduce that?

Reply:
for toplevel keys with single sourcekey candidate, I think it won't affect
wdyt

Reply:
Agreed